From 8b2aa9bdb7740cea373cbdf09e90fac58daa0ff0 Mon Sep 17 00:00:00 2001
From: "Sodhi, VunnyX" <vunnyx.sodhi@intel.com>
Date: Mon, 15 Jun 2015 12:33:39 +0530
Subject: [PATCH] ASoC: Intel: Increasing VTSV NET and VTSV Grammar file size
 for multiphrase support

Changes are done to increase file size for LPAL model to 384KB
in DDR IMR region by increasing NET to 352kB of vtsv_net.bin and
GRAMMAR to 32kB of vtsv_grammar.bin for multiphrase support.

Change-Id: Id62be709c1ef346650ba30290cf2b63af800dbee
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-36748
Tracked-On: https://jira01.devtools.intel.com/browse/GMINL-12449
Signed-off-by: Sodhi, VunnyX <vunnyx.sodhi@intel.com>
---
 sound/soc/intel/sst/sst_stream.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/sound/soc/intel/sst/sst_stream.c b/sound/soc/intel/sst/sst_stream.c
index afb13c6dac37..3d1fd7cf356d 100644
--- a/sound/soc/intel/sst/sst_stream.c
+++ b/sound/soc/intel/sst/sst_stream.c
@@ -808,9 +808,9 @@ int sst_format_vtsv_message(struct intel_sst_drv *ctx,
 	return 0;
 }
 
-/* Total 127 KB is allocated for vtsv net/grammar file */
-#define VTSV_NET_MAX_SIZE 0x1E000	/* max net file size: 120 KB */
-#define VTSV_GRAMMAR_MAX_SIZE 0x1C00 /* max grammar file size: 7 KB */
+/* Total 384 KB is allocated for vtsv net/grammar file */
+#define VTSV_NET_MAX_SIZE 0x58000	/* max net file size: 352 KB */
+#define VTSV_GRAMMAR_MAX_SIZE 0x8000 /* max grammar file size: 32 KB */
 
 int sst_cache_vtsv_libs(struct intel_sst_drv *ctx)
 {
-- 
1.9.1

